code/modules/projectiles/gun_attachables.dm

```diff
@@ -2073,6 +2073,9 @@ Defined in conflicts.dm of the #defines folder.
 	burn_level = BURN_LEVEL_TIER_5
 	burn_duration = BURN_TIME_TIER_2
 
+/obj/item/attachable/attached_gun/flamer/integrated/advanced//different name, same stats, intended to be used as a removable version
```

Comment:
this shouldn't be subtyped off "integrated", pathing should be like advanced and advanced/integrated